### PR TITLE
Fix/removing unused codes

### PR DESCRIPTION
**Pull request by xckitahara**
Remove the code that will not be executed.
- src/utils
- Unused commands
- Classes that will no longer be used due to the above deletion
- Other Unnecessary Codes
  - local variable
  - imports

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -22,9 +22,6 @@ repositories {
 sourceSets {
     utils {
```
Comment (alumi) [inline build.gradle]: Remove `utils { ... }` and all related configurations

**Review by alumi — COMMENTED** [review]
Thank you for tidying up the code! Basically the changes look good 👍 

Would you check if the removed method calls don't have side effects?
`T variable = obj.method()` can be removed iff `variable` is not used and `method` has no side effects.

e.g. https://github.com/genome-rcast/karkinos/pull/14/files#diff-bfe051ab5c91d4a1e2c53b9528f26571L196 can be removed as `getPloidyFlg` has no side effects.

**Comment by xckitahara** [comment]
I thought I removed any unused variables from the code inspection, but was there anything else left?
The example has also been deleted.

**Review by alumi — COMMENTED** [review]
I meant I want you to be sure that the deletions are safe.
Unused variables cannot be removed only because they are unused.

Apart from that, those configurations should be deleted, too
https://github.com/genome-rcast/karkinos/blob/44a61e400fa812c94be9216fdf374b13470fddff/build.gradle#L30-L34
https://github.com/genome-rcast/karkinos/blob/44a61e400fa812c94be9216fdf374b13470fddff/build.gradle#L58

**Comment by xckitahara** [comment]
Sorry. I understand.
I checked and I think there is no such thing.

The build.gradle has been fixed.

**Review by alumi — APPROVED** [review]
Thank you for your confirmation! LGTM 👍

**Pull request merged**